tests/std/tests/GH_003003_format_fix_decimal_point_and_trailing_zero/env.lst

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+RUNALL_INCLUDE ..\usual_latest_matrix.lst

```

Comment:
The ...mat_"fix"_deci... in the folder name seems redundant...

Reply:
Perhaps we should use concepts_20_matrix.lst which skips EDG. IIUC this bug needs to be and will be fixed in C++20 mode.
```suggestion
RUNALL_INCLUDE ..\concepts_20_matrix.lst
```